library/cisco_ucs_ntp.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+
+from ansible.module_utils.basic import *
+
+
+DOCUMENTATION = '''
+---
+module: cisco_ucs_ntp
+short_description: configures ntp on a cisco ucs server
+version_added: 0.9.0.0
+description:
+   -  configures ntp on a cisco ucs server
+Input Params:
+    name:
+        description: ntp server ip address or hostname
+        required: True
+    descr:
+        description: ntp server description
+        required: False
+
+requirements: ['ucsmsdk', 'ucsm_apis']
+author: "Rahul Gupta(ragupta4@cisco.com)"
+'''
+
+
+EXAMPLES = '''
+- name:
+  cisco_ucs_ntp:
+    name:
```

Comment:
example should be complete.. `name` is a required param and is empty in the example.